x/scheduler/types/mocks/BankKeeper.go

```diff
@@ -0,0 +1,50 @@
+// Code generated by mockery v2.38.0. DO NOT EDIT.
```

Comment:
Why are these new? Are they used anywhere?

Reply:
deleted unused mock files and pushed.

Reply:
Resolved.